vmupdate/agent/source/plugins/apt_keep_old_conffiles.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2024  Piotr Bartman <prbartman@invisiblethingslab.com>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+# USA.
+
+APT_CONF = "/etc/apt/apt.conf.d/01qubes-update"
```

Comment:
This will change also behavior for interactive calls later one, which isn't a good idea.
Maybe you can use `apt-get -o ..`, or `apt-get -c` (with some temp file)? Or equivalent via APT API?